src/openzaak/components/autorisaties/forms.py

```diff
@@ -428,8 +437,31 @@ def save(self, commit=True):
         if not versions_equivalent(old_version, new_version):
             send_applicatie_changed_notification(self.applicatie, new_version)
 
+    def clean(self):
+        # validate overlap zaaktypen between different auths
+        self._validate_overlapping_types()
+
+    def _validate_overlapping_types(self):
+        scope_types = {}
+        for form in self.forms:
+            if form.cleaned_data:
+                types = set(form.get_types() or [])
+                scopes = form.cleaned_data["scopes"]
+                types_field = COMPONENT_TO_FIELDS_MAP[form.cleaned_data["component"]][
+                    "types_field"
+                ]
+                for scope in scopes:
+                    previous_types = scope_types.get(scope, set())
+                    if previous_types.intersection(types):
+                        raise ValidationError(
+                            _("{} should not overlap scopes in one Applicatie").format(
```

Comment:
```suggestion
                            _("{field} may not have overlapping scopes.").format(
```

1. Phrasing fixed
2. Always use keyword arguments in translatable strings - different languages have different order of nouns/verbs, having named variables makes this easier to achieve.